enemy/enemy_alt.gd

```diff
@@ -11,24 +12,26 @@ signal died
 @export var hit_particle_color: Color
 @export var attack_sfx: AudioStreamPlayer2D
 @export var attack_frame_index: int = 2
+@export var health_bar: HealthBar
 
-@export var health: int = 2
+@export var max_health: int = 1
 @export var damage: int = 1
 @export var speed: float = 100.0
 @export var coin_reward: int = 1
 
+var _health: int
 var _attack_position = Vector2.INF
 var _state = 'idle'
 
 signal entered_attack_hit_frame
 
 
 func is_alive() -> bool:
-	return health > 0
+	return _health > 0
 
 
 func set_difficulty(difficulty: float) -> void:
-	health += ceil(difficulty)
+	max_health += ceil(difficulty)
 	damage += ceil(difficulty * 0.5)
 	speed += ceil(difficulty * 0.5)
 	coin_reward += floor(difficulty * 1.5)
```

Comment:
we don't use this function yet. was added in a previous PR. could use it in the `enemy_spawn` script

Reply:
removed `set_difficulty`

Reply:
removed `set_difficulty`